src/structures/vroom/input/input.h

```diff
@@ -29,6 +29,9 @@ using Servers = std::unordered_map<std::string, Server>;
 
 class VRP;
 
+constexpr unsigned DEFAULT_EXPLORATION_LEVEL = 5;
+constexpr unsigned DEFAULT_THREADS_NUMBER = 4;
```

Comment:
Probably a matter of taste but it may be more clear to put those in `src/structures/typedefs.h` where we already have various constexpr named constants used across the project.

Reply:
Sure.